### ISSUE TITLE
Specification of table names in DLP Dataset for Sensitive Tag Column

### ISSUE THREAD

**Issue by gotoyuto-ca** [GoogleCloudPlatform/datacatalog-tag-engine#121]
It seems that the table name you want to tag and the table name of the DLP dataset need to be the same, when running [Sensitive Tag Column](https://github.com/GoogleCloudPlatform/datacatalog-tag-engine/blob/cloud-run/docs/ui_guide.md#sensitive_column). 

However, there is no explanation for table names in DLP Datasets.

If this is the specification, it should be added to `ui_guide.md`.

https://github.com/GoogleCloudPlatform/datacatalog-tag-engine/blob/b64520a4780e66e72215efd5cf530e3f87b61059/docs/ui_guide.md?plain=1#L93-L95

**Comment by shirleycohen** [comment]
Yes, that's true. Are you saying that for you the table names don't match up? (i.e. the name of the table you want to tag does is different from the name of the table written by DLP?).
